code/src/components/Thought.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { formatDistance } from 'date-fns'
+import parseISO from 'date-fns/parseISO'
+
+const Thought = ({thought, handleLikes}) => {
+    console.log(thought)
+
+    const timePosted = formatDistance(parseISO(thought.createdAt), new Date(), {addSuffix:true})
```

Comment:
Really interesting solution - I got an error for date-fns since date was a string, so I used moment instead, but since it is depreciated I should look into this instead :]